Examples/Detectors/DD4hepDetector/compact/Calorimeter/Calorimeter_LAR.xml

```diff
@@ -0,0 +1,142 @@
+        <constant name="lay3_env_dz" value="2.5*m"/>
+        <constant name="lay3_env_z" value="0*m"/>
+        <!--Dimensions of Layer 4 -->
+        <constant name="lay4_env_rmin" value="1422*mm"/>
+        <constant name="lay4_env_rmax" value="1429*mm"/>
+        <constant name="lay4_env_dz" value="2.5*m"/>
+        <constant name="lay4_env_z" value="0*m"/>
+        <!--Dimensions of Layer 5 -->
+        <constant name="lay5_env_rmin" value="1429*mm"/>
+        <constant name="lay5_env_rmax" value="1436*mm"/>
+        <constant name="lay5_env_dz" value="2.5*m"/>
+        <constant name="lay5_env_z" value="0*m"/>
+        <!--Dimensions of Layer 6 -->
+        <constant name="lay6_env_rmin" value="1436*mm"/>
+        <constant name="lay6_env_rmax" value="1443*mm"/>
+        <constant name="lay6_env_dz" value="2.5*m"/>
+        <constant name="lay6_env_z" value="0*m"/>
+        <!--Dimensions of Layer 7 -->
+        <constant name="lay7_env_rmin" value="1443*mm"/>
+        <constant name="lay7_env_rmax" value="1450*mm"/>
+        <constant name="lay7_env_dz" value="2.5*m"/>
+        <constant name="lay7_env_z" value="0*m"/>
+
+    </define>
+
+    <display>
+        <vis name="Beryllium" r="0.8" g="0.8"  b="0.8" alpha="0.9" showDaugthers="true" visible="true" drawingStyle="solid"/>
+        <vis name="Kapton" r="0.1328125" g="0.55859375" b="0.02734375" alpha="1" showDaugthers="true" visible="true" drawingStyle="solid"/>
+        <vis name="violet" r="0.4" g="0." b="0.4" alpha="1" showDaugthers="true" visible="true" drawingStyle="solid"/>
+        <vis name="red" r="0.6" g="0." b="0." alpha="1" showDaugthers="true" visible="true" drawingStyle="solid"/>
+        <vis name="yellow" r="0.65" g="0.65" b="0." alpha="1" showDaugthers="true" visible="true" drawingStyle="solid"/>
+        <vis name="orange" r="0.65" g="0.35" b="0." alpha="1" showDaugthers="true" visible="true" drawingStyle="solid"/>
+        <vis name="blue" r="0." g="0." b="0.5" alpha="1" showDaugthers="true" visible="true" drawingStyle="solid"/>
+        <vis name="green" r="0." g="0.5" b="0." alpha="1" showDaugthers="true" visible="true" drawingStyle="solid"/>
+        <vis name="grey" r="0.7" g="0.6" b="0.5" alpha="1" showDaugthers="true" visible="true" drawingStyle="solid"/>
+        <vis name="bluetrans" alpha="0.5" r="0.0" g="0.0" b="0.5" showDaughters="true" visible="true"/>
+    </display>
+
+    <detectors>
+      <detector id="0" name="BeamPipe" type="ODDCylinder" vis="Beryllium">
+            <beampipe/>
+            <tubs name="BeamPipeCylinder" rmin="bp_rmin" rmax="bp_rmax" dz="bp_length" material="Beryllium"/>
+      </detector>
+      <detector id="1" name="Calorimeter" type="Calorimeter" beampipe="false" vis="orange">
+            <dimensions name="CalorimeterShape" rmin="Calo_env_rmin" rmax="Calo_env_rmax" dz="Calo_env_dz" z="Calo_env_z"/>
+            <tubs name="Entrance_1" rmin="Calo_env_rmin" rmax="lay1_env_rmin-0.01*mm" dz="lay1_env_dz" material="Air"/>
```

Comment:
Is the `rmax` a trick to build an object with 0 thickness?

Reply:
No here the envelop was here for similar reason as for the previous comment. I also removed it.